src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -22,12 +22,12 @@
           </a>
         </li>
         {% else %}
-          {% if crumb.text == "Health Care" %}
+          {% if crumb.text == "Health care" %}
             {% assign crumbPath = "/health-care" %}
           {% else %}
             {% assign crumbPath = crumb.url.path %}
           {% endif %}
-          {% if crumb.text != "Get Benefits" and crumb.text != "Manage Benefits" and crumb.text != "More Resources" %}
+          {% if crumb.text != "Get benefits" and crumb.text != "Manage benefits" and crumb.text != "More resources" %}
         <li>
           {% if crumbPath != empty %}
```

Comment:
Why isn't this line preventing the `<a>` tag from being generated?

Reply:
Because the data is shaped funny -

![image](https://user-images.githubusercontent.com/1915775/56171588-dc38b780-5fb3-11e9-930b-2ced71941229.png)

I haven't worked with this, btw. I traced this back just now.

Reply:
Yes, I assume someone went in and sentence cased those menu items. 

But I meant line 32 (`{% if crumbPath != empty %}`). Shouldn't that prevent `<a href="{{crumbPath}}"` from being processed?

Reply:
Ah yeah I saw you meant and remove my response just before your followed up...I'm not sure.

Running locally to find out...

Reply:
Unfortunately @beeyayjay I'm not sure, but I can't spend any more time on this and I'm holding up other engineers. `{% if crumbPath != "" %}` does work though, so it seems like something with the `empty` literal.

@jbalboni Can you think of any reason this might be the case?